File: lab2-3.py
while True:
 x = int(input("Give a number between 3 and 9:"))
 if x < 3 or x > 9:
    print('Invalid input. Please enter a number between 3 and 9')
    continue
 break

old = 0

while old <= x:

    for y in range(1, old +1, 1):
        print(y)
    
    old += 1

File: lab2_3.cpp
#include <iostream>
#include <string>
using namespace std;

int main () {

    int n;
    cout << "Enter a number between 3 - 9:";
    cin >> n;

    string s = "";
    for (int i = 1; i < n + 1; i++) {
        s += to_string(i);
        cout << s << endl;
    }
    for (int i = n; i >= 1; i--) {
        s.pop_back();
        cout << s << endl;
    }
    return 0;

}